gridded_etl_tools/utils/convenience.py

```diff
@@ -664,4 +661,25 @@ def standardize_longitudes(cls, dataset: xr.Dataset) -> xr.Dataset:
             longitude=(((dataset.longitude + 180) % 360) - 180)
         )
         # After converting, the longitudes may still start at zero. This reorders the longitude coordinates from -180 to 180 if necessary.
-        return dataset.sortby(["latitude", "longitude"])
\ No newline at end of file
+        return dataset.sortby(["latitude", "longitude"])
+
+    @classmethod
+    def span_to_timedelta(cls) -> np.timedelta64:
+        """
+        Map a dataset's string time span to a corresponding numpy timedelta64 object
+        NOTE only valid for datasets with regular update cadences.
+        See `irregular_update_cadence` attribute for handling irregular cadence.
+
+        Returns
+        -------
+        np.timedelta64
```

Comment:
We should drop the type since the type is annotated. See function_with_pep484_type_annotations at https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html